lib/src/location_picker_field.dart

```diff
@@ -119,6 +129,7 @@ class __BaseFormFieldState<T> extends State<_BaseFormField<T>> {
                 : effectiveDecoration.suffixIcon);
 
         return InkWell(
+          borderRadius: widget.borderRadius,
```

Comment:
if this for clipping may be use `ClipPath` is best fit as the input border may not be totally round or has a custom shape 
```dart
ClipPath(
      clipper: ShapeBorderClipper(effectiveDecoration.border),
    );
```